cmd/imagec/imagec.go

```diff
@@ -673,6 +674,13 @@ func main() {
 		log.Debugf("Running standalone")
 	}
 
+	// track whether the image is from a custom registry or not
+	registryURL := fmt.Sprintf("%s/", options.registry)
```

Comment:
Oh, I see;  you're testing for equivalency.  Consider using the url package and the `String()` method to compare against.  Up to you.